neuromation/cli/root.py

```diff
@@ -19,6 +20,11 @@
 
 TEXT_TYPE = ("application/json", "text")
 
+HEADER_TOKEN_PATTERNS = [
+    re.compile(rf"({auth_scheme})\s+([^ ]+\.[^ ]+\.[^ ]+)")
+    for auth_scheme in ("Bearer", "Basic", "Digest", "Mutual")
```

Comment:
I took the most popular auth types. Should we include others ([mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#Authentication_schemes) recommends also `HOBA`, `AWS4-HMAC-SHA256`.). What comes to other auth schemes: 
- `Bearer`: we use it
- `Basic`, `Digest` -- we might use it one day and thus current problem will be solved for these cases.  However, for sake of performance, we can remove these so-far-unsopported cases for now.
- `Mutual` -- as far as I know, we *will* use mutual authentication between client and server in future.